src/admin/items/components/DepublishItemModal/DepublishItemModal.tsx

```diff
@@ -0,0 +1,219 @@
+
+			// When we unpublish an item, it cannot be the replacement for any other items
+			await RelationService.deleteRelationsByObject(
+				'item',
+				RelationType.IS_REPLACED_BY,
+				item.uid
+			);
+
+			if (depublishType === 'depublish_with_reason') {
+				// Remove after https://meemoo.atlassian.net/browse/DEV-1140
+				ToastService.danger(
+					t('Depubliceren met reden is nog niet geïmplementeerd (AVO-699)'),
+					false
+				);
+				return;
+				// await ItemsService.setItemDepublishReason(item.external_id, reasonHtml);
+			} else if (depublishType === 'depublish_with_replacement' && replacementExternalId) {
+				const replacementItem: Avo.Item.Item | null = await ItemsService.fetchItemByExternalId(
+					replacementExternalId
+				);
+				if (!replacementItem) {
+					ToastService.danger(
+						t('Het bepalen van de id van het vervang item is mislukt'),
+						false
+					);
+					return;
+				}
+
+				// Add the replacement as instructed by the user
+				await RelationService.insertRelation(
+					'item',
+					item.uid,
+					RelationType.IS_REPLACED_BY,
+					replacementItem.uid
+				);
+			}
+			ToastService.success(t('Het item is gedepubliceerd'), false);
+			handleClose();
+		} catch (err) {
+			console.error(
+				new CustomError('Failed to depublish item', err, {
+					item,
+					depublishType,
+					reason,
+					replacementExternalId,
+				})
+			);
+			ToastService.danger(t('Het depubliceren is mislukt'), false);
+		}
+	};
+
+	return (
+		<Modal
+			isOpen={isOpen}
+			title={t('Item depubliceren')}
+			size="medium"
+			onClose={handleClose}
+			className="m-depublish-modal"
+		>
+			<ModalBody>
+				<Form>
+					<FormGroup label={t('Hoe depubliceren?')}>
+						<Select
+							options={[
+								{
+									label: t('Enkel depubliceren'),
+									value: 'depublish',
+								},
+								{
+									label: t('Depubliceren met reden'),
+									value: 'depublish_with_reason',
+								},
+								{
+									label: t('Depubliceren en vervang item aanduiden'),
+									value: 'depublish_with_replacement',
+								},
+							]}
+							value={depublishType}
+							onChange={setDepublishType as (value: string) => void}
+						/>
+					</FormGroup>
+					{depublishType === 'depublish_with_reason' && (
+						<FormGroup label={t('Reden tot depubliceren:')}>
+							<WYSIWYGWrapper
+								placeholder={t(
+									'Geef een reden waarom dit item gedepubliceerd wordt'
+								)}
+								state={reason || undefined}
+								onChange={setReason}
+							/>
+						</FormGroup>
+					)}
+					{depublishType === 'depublish_with_replacement' && (
+						<FormGroup label={t('Selecteer item ter vervanging:')}>
+							<ContentPicker
+								initialValue={
+									replacementExternalId
+										? {
+												type: 'ITEM',
+												value: replacementExternalId,
+										  }
+										: undefined
+								}
+								onSelect={pickerItem =>
+									setReplacementExternalId(pickerItem ? pickerItem.value : null)
```

Comment:
Is `pickerItem ? pickerItem.value : null` dan niet gewoon `get(pickerItem, 'value', null)`?